### PR TITLE
Add ScalarDB Cluster (BYOL) in Marketplace guides

### PR DESCRIPTION
**Pull request by kota2and3kan**
## Description

Now, we start providing `ScalarDB Cluster (BYOL)` in AWS/Azure Marketplace.

* [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-alcwrmw6v4cfy) (Note: It is still in progress. Not published yet.)
* [Azure Marketplace](https://azuremarketplace.microsoft.com/ja-jp/marketplace/apps/scalarinc.scalardb?tab=PlansAndPrice) (Note: We already published.)

So, I updated each Marketplace Guide.

## Related issues and/or PRs

N/A

## Changes made

* Added sample configurations and commands when we use ScalarDB Cluster.
* Marked ScalarDB Server as `Deprecated` and removed sample configurations and commands about ScalarDB Server.

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

Some documents still have descriptions of `ScalarDB Server`. So, I will mark `ScalarDB Server` as `Deprecated` in all documents in another PR.

### CODE REVIEW

**Review by josh-wong — APPROVED** [review]
LGTM! Thank you!🙇‍♂️

File: docs/AzureMarketplaceGuide.md [hunk 1]
```diff
@@ -33,10 +33,10 @@ Note that **Company** is not required, but please enter it.
    You need several container images to run Scalar products on Kubernetes, but Azure Marketplace copies only one container image at a time. So, you need to subscribe to several software plans (repeat subscribe operation) as needed.
    * Container images that you need are the following.
         * ScalarDB
+            * ScalarDB Cluster (BYOL)
```
Comment (komamitsu) [inline docs/AzureMarketplaceGuide.md:36]: 🎉

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**